src/it/scala/nl/wbaa/testkit/s3sdk/sdkSetup.scala

```diff
@@ -0,0 +1,30 @@
+package nl.wbaa.testkit.s3sdk
+
+import com.amazonaws.ClientConfiguration
+import com.amazonaws.auth.{AWSStaticCredentialsProvider, BasicAWSCredentials}
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+import com.typesafe.config.ConfigFactory
+
+
+trait sdkSetup {
+  import sdkSetup._
+
+  def getAmazonS3(awsSignerType: String): AmazonS3 = {
+    val cliConf = new ClientConfiguration()
+    cliConf.setMaxErrorRetry(1)
+    cliConf.setSignerOverride(awsSignerType)
+
+    AmazonS3ClientBuilder
+      .standard()
+      .withClientConfiguration(cliConf)
+      .withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials("accesskey", "secretkey")))
+      .withEndpointConfiguration(new EndpointConfiguration(s"http://127.0.0.1:$proxyPort", "us-west-2"))
+      .build()
+  }
+}
+
+object sdkSetup {
+  private val configProxy = ConfigFactory.load().getConfig("proxy.server")
+  val proxyPort: Int = configProxy.getInt("port")
+}
```

Comment:
I'm mildly perplexed as to why this stuff is in here?

Normally our tests should be testing stuff from `main/` but these are exploring the AWS SDK or something?

Reply:
This there since our proxy server will get started on the port specified in the `application.conf`. In our tests sdk we'll have to connect to that